csharp/ql/src/utils/modelgenerator/internal/CaptureModels.qll

```diff
@@ -361,7 +361,29 @@ module ModelGeneratorInput implements ModelGeneratorInputSig<Location, CsharpDat
     c.isDelegateCallReturn() and result = "ReturnValue"
   }
 
-  predicate partialModel = ExternalFlow::partialModel/6;
+  string partialModelRow(Callable api, int i) {
+    i = 0 and ExternalFlow::partialModel(api, result, _, _, _, _) // package
+    or
+    i = 1 and ExternalFlow::partialModel(api, _, result, _, _, _) // type
+    or
+    i = 2 and ExternalFlow::partialModel(api, _, _, result, _, _) // extensible
+    or
+    i = 3 and ExternalFlow::partialModel(api, _, _, _, result, _) // name
+    or
+    i = 4 and ExternalFlow::partialModel(api, _, _, _, _, result) // parameters
+    or
+    i = 5 and result = "" and exists(api) // ext
```

Comment:
So replacing the conjunction with disjuncts (and index) and a subsequent concat works because the values in each column is a *function* of the `api`. If we ever extend the model generator to cater for attributes/annotation, this might no longer hold (but if that ever happens then most likely `ext` will be removed from the prefix/partial part).